src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -1225,7 +1225,18 @@ namespace
 
             QVBoxLayout* layout = new QVBoxLayout();
             container()->setLayout(layout);
-
+            #ifdef APPLESEED_WITH_EMBREE
+            create_system_use_embree_settings(layout);
+            create_direct_link("use_embree", "use_embree", "false");
+            if (config.get_parameters().strings().exist("use_embree"))
```

Comment:
Can we reduce this whole block to simply

    set_widget("use_embree", 
        config.get_parameters().get_optional<bool>("use_embree", false");

?

Reply:
In fact, since you've created a direct link just above, the `load_directly_linked_values()` call below should already take care of all this, so this code appears redundant to me.